pages/establishment/update/views/index.jsx

```diff
@@ -74,8 +74,7 @@ class Repeat extends Component {
         authorisations: authorisations.map((authorisation, i) => {
           if (i === index) {
             return {
-              ...authorisation,
-              ...model
+              ...authorisation
```

Comment:
This isn't doing anything at all to the state. Either `i === index` and it returns the authorisation, or it doesn't and it returns... the authorisation.

There's no update.

Reply:
Removed onFieldsetChange - we do not need it.